pages/learn/integration/pre-built-integrations/ftp-edi-message-to-salesforce-opportunity.js

```diff
@@ -206,7 +206,7 @@ export default function Learn({ samples, content }) {
           content="ballerina, learn, documentation, docs, programming language"
         />
         <link rel="shortcut icon" href="/img/favicon.ico" />
-        <title>FTP EDI message to Salesforce Opportunity</title>
+        <title>FTP EDI messages to Salesforce opportunities integration</title>
```

Comment:
Singular form sounds relevant to the usecase.
Also not sure we need to use "integration" in this.

Reply:
+1 for @chathurace's suggestion
Also unlike in other pages we specifically say it is to opportunity, so having integration seems too wordy.

Reply:
Fixed.